PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -118,10 +123,13 @@ struct HfCandidateCreatorXic0Omegac0 {
   Configurable<float> lambdaMassWindow{"lambdaMassWindow", 0.0075, "Distance from Lambda mass"};
   // cascade cuts
   Configurable<double> massToleranceCascade{"massToleranceCascade", 0.01, "Invariant mass tolerance for cascade"};
+  Configurable<double> massToleranceCascadeRej{"massToleranceCascadeRej", 0.01, "Invariant mass tolerance for rejected Xi"};
   // for KF particle operation
   Configurable<int> kfConstructMethod{"kfConstructMethod", 2, "KF Construct Method"};
   Configurable<bool> kfUseV0MassConstraint{"kfUseV0MassConstraint", false, "KF: use Lambda mass constraint"};
   Configurable<bool> kfUseCascadeMassConstraint{"kfUseCascadeMassConstraint", false, "KF: use Cascade mass constraint"};
+  Configurable<bool> constrainToPv{"constrainToPv", false, "Constrain particles to PV"};
+  Configurable<bool> constrainToMother{"constrainToMother", false, "Constrain daughter particles to mother particles"};
```

Comment:
specify that these are configurables for the KF, as the previous ones (both in the name of the variable, and in the comment)
You might even consider to add `configurableGroups` to make it more organised, see e.g.  
https://github.com/AliceO2Group/O2Physics/blob/0882ed52c194504c30aea83b48ca377ac8fdba65/PWGHF/D2H/TableProducer/dataCreatorCharmResoReduced.cxx#L130-L136

Reply:
Thanks! You can anyway consider to use the configurable groups (they are also very useful to reduce the number of data members and avoid hitting the limit ;-)
It can also be done later in a second PR